R/data.table.R

```diff
@@ -1739,22 +1739,48 @@ replace_dot_alias = function(e) {
         GForce = FALSE
       } else {
         # Apply GForce
+        noCall_noVars = function(expr) (!is.call(expr) || length(all.vars(expr, max.names=1L, unique=FALSE))==0L) && !dotN(expr)
```

Comment:
still not a fan of the name here... WDYT about `is_constant_ish()`? or `no_call_nor_vars()`?

Reply:
No hard feelings about the name, but would rather go with `is_constantish` or `is_constantIsh` instead of the double dash.

Reply:
`is_constantish` works for me!